packages/e2e-tests/src/specs/dashboard/myStories.js

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import {
+  createNewStory,
+  publishStory,
+  visitDashboard,
+  insertStoryTitle,
+} from '@web-stories-wp/e2e-test-utils';
+
+describe('Stories Dashboard', () => {
+  it('should delete story', async () => {
+    //need to create new story as all stories are deleted before test runs.
+    await createNewStory();
+    await insertStoryTitle('Delete this story');
+    await publishStory();
+    await visitDashboard();
```

Comment:
you can simplify this since it's the same series before each action you're testing by adding 
```
beforeEach(async () => {
    await createNewStory();
    await insertStoryTitle('Delete this story');
    await publishStory();
    await visitDashboard();
}); 
```

then you can don't need to have it within each individual test.

I know you're naming the story specific to the action but if you wanted to, you could make it something generic, store it as a constant and then reference that value where you're finding by labels with some interpolation to decrease potential typos. 

So like: 

`const storyName = 'Test Story';`

`'[role="listitem"][aria-label="Details about ${storyName}"]'`